src/pattern-library/components/patterns/ContainerPatterns.js

```diff
@@ -182,7 +182,7 @@ export default function ContainerPatterns() {
                 Show example
               </LabeledButton>
               <div
-                className="hyp-u-overlay"
+                className="fixed top-0 left-0 bottom-0 right-0 bg-black/50 z-5"
```

Comment:
Yes, this is wordy. This pattern may bear abstraction in future — an `Overlay` component? something? — but the `.hyp-u-overlay` class is not part of the CSS API and this styling is currently only used in pattern-library examples. So I have opted, for now, for the "unrolled" representation using TW utility classes to avoid any premature abstraction.